src/instruments/src/EFB/Components/Progress/Progress.tsx

```diff
@@ -107,39 +108,46 @@ export const ProgressBar: React.FC<ProgressBarProps> = ({
         alignItems: labelAlignment === 'outside' ? 'center' : 'initial',
     };
 
-    const checkOrientation = () => {
-        if (vertical) {
-            return 'horizontal-progress-bar';
+    const getBarStyle = () => {
+        if (completedBarBeginValue && completedBarEnd && completionValue) {
```

Comment:
**thought:** We might want this feature (green bar) disabled in other progress bars. As this PR adds the feature to the generic component, it should be optional.

**suggestion:** Add a prop to enable this.

Reply:
good idea